lib/glimesh_web/live/user_live/components/social_buttons.ex

```diff
@@ -1,13 +1,15 @@
 defmodule GlimeshWeb.UserLive.Components.SocialButtons do
   use GlimeshWeb, :live_view
 
+  alias Glimesh.Socials.Sanitizer
+
   @impl true
   def render(assigns) do
     ~L"""
     <%= if @twitter_social  do %>
     <li rel="ugc" class="list-inline-item" data-toggle="tooltip"
         title="<%= gettext("Linked Twitter Account @%{username}", username: @twitter_social.username) %>">
-        <a href="https://twitter.com/<%= @twitter_social.username %>" target="_blank">
+        <a href="https://twitter.com/<%= Sanitizer.sanitize(@twitter_social.username) %>" target="_blank">
```

Comment:
Isn't this essentially sanitizing it twice? Shouldn't we migrate all the existing records to be safe?

Reply:
Thats correct, this is just a monkey patch to correct the records that exist. 

I can write a data migration if preferred; do you prefer running these types of non-schema migrations on an iex in prod or making a db migration? (Or any other way? I know people usually have a preference)